lib/personnummer.dart

```diff
@@ -57,7 +98,7 @@ class Personnummer {
     String check = match[7];
 
     if (check.isEmpty) {
-      return map;
+      throw new PersonnummerException();
```

Comment:
Maybe a message in the exception, so that the user knows what they did wrong!

Reply:
The user get the message `Invalid swedish social security number`, not sure if we need a parsing specific message.